components/widget/match/summary/widget_match_summary_characters.lua

```diff
@@ -15,11 +15,15 @@ local HtmlWidgets = Lua.import('Module:Widget/Html/All')
 local Div = HtmlWidgets.Div
 local Character = Lua.import('Module:Widget/Match/Summary/Character')
 
+local BASE_SIZE = 24 -- From brkts-champion-icon in Brackets.less
+local HOVER_MODIFIER = 2.5 -- From brkts-champion-icon in Brackets.less
+
 ---@class MatchSummaryCharacters: Widget
 ---@operator call(table): MatchSummaryCharacters
 local MatchSummaryCharacters = Class.new(Widget)
 MatchSummaryCharacters.defaultProps = {
 	flipped = false,
+	size = BASE_SIZE * HOVER_MODIFIER,
```

Comment:
Do we need to round this somehow? Don't know whether mediawiki can handle decimal sizes correctly